crates/network-scheduler/src/cli.rs

```diff
@@ -43,6 +43,7 @@ pub struct Config {
     pub signature_refresh_interval: Duration,
     pub replication_factor: usize, // this is minimum
     pub dynamic_replication: bool,
+    pub dynamic_replication_factor: f64,
```

Comment:
Shouldn't it have a default value?

Reply:
No, I will push config update together with code update.